parsl/tests/integration/test_channels/test_ssh_errors.py

```diff
@@ -25,17 +25,6 @@ def test_error_2():
         raise Exception("Expected SSException, got :{0}".format(e))
 
 
-def test_error_3():
```

Comment:
Why is this being removed? Can we not just rename it?

Reply:
This test doesn't ever run - it's shadowed by the subsequent definition. This PR was basically removing all such code as being effectively commented out and dead.

The version that runs on `login.mcs.anl.gov` was introduced in  64fa6ab64b on  Oct 12 17:24:25 2017, but then shadowed a few minutes later by the `edison` version in e6190c833f.

I'd be fine with the `login.mcs.anl.gov` test being re-enabled by renaming the method, if that's still a sensible test - these tests are pretty specific to @yadudoc 's accounts so I don't really have the ability to verify re-enabling test_error_3.